src/to-string.cpp

```diff
@@ -66,6 +66,28 @@ const char * get_string( rs2_stream value )
 #undef CASE
 }
 
+char const * get_abbr_string( rs2_stream value)
+{
+    switch( value )
+    {
+    case RS2_STREAM_ANY: return "any";
```

Comment:
Why use only lower case here and all other start with uppercase?
`Conf` for exanple?
Consider aligning it

Reply:
I wanted it to stand out, but OK: changed.